04/app.js

```diff
@@ -58,7 +84,7 @@ class App extends React.Component {
         e.preventDefault();
 
         const { firstName, lastName } = this.state;
-        if(firstName && lastName) {
+        if (firstName && lastName) {
```

Comment:
Przydałoby się dodać ten komunikat dla użytkownika - najlepiej w oparciu o `state` ;P

Reply:
Przeoczyłem tę linijkę :o. Dzięki, że zwróciłeś na to uwagę – wydaje mi się, że znalazłem ciekawy sposób na wykorzystanie `state `w tym przykładzie :D.